src/frontend/magnify/src/components/profile/IdentityForm/IdentityForm.tsx

```diff
@@ -86,8 +86,8 @@ export default function IdentityForm() {
 
   return (
     <Formik
+      enableReinitialize={true}
```

Comment:
No risk of blinking as mentionned in next commit for `RoomConfig` ?

Reply:
Nope. In RoomConfig, it's because we submit after 800ms of inactivity in the form. There is a submit button